command.php

```diff
@@ -22,7 +22,9 @@ class WooCommerce_Blocks_Testing_Environment extends WP_CLI_Command {
 	 * ## OPTIONS
 	 *
 	 * [--blocks[=<version>]]
```

Comment:
What do you think about the following change:

```suggestion
	 * [--blocks[=<version|url>]]
```

AFAICS, this change does not affect the code that handles installing the desired WooCommerce Blocks version or link. This change would simply improve the UX when looking up available commands using the terminal, e.g. when running

```sh
wp help woo-test-environment setup
```

<img width="1680" alt="Screenshot 2022-07-08 at 11 32 59" src="https://user-images.githubusercontent.com/3323310/177963395-d1e09888-e0b4-4ee2-93d9-9025fd764f1a.png">